cluster/images/kubelet/Dockerfile

```diff
@@ -1,8 +1,13 @@
-FROM centos
-ADD kubelet /kubelet
+FROM debian:jessie
+
+COPY kubelet /kubelet
 RUN chmod a+rx /kubelet
-RUN cp /usr/bin/nsenter /nsenter
 
+COPY kubelet-runner.sh /kubelet-runner.sh
+RUN chmod a+rx /kubelet-runner.sh
+
```

Comment:
There are other requirements for hyperkube, e.g. we generate certs there. I'll add those that I believe are really needed.